src/poll.rs

```diff
@@ -53,7 +119,32 @@ pub fn poll(fds: &mut [PollFd], timeout: libc::c_int) -> Result<libc::c_int> {
     Errno::result(res)
 }
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
+/// `ppoll()` allows an application to safely wait until either a file
```

Comment:
`poll` returns also on signals but the caller do not control what. But I agree that something like "Like `poll()` but let you precise what signals may interrupt it" is enough. I'll change that.